packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -1861,6 +1861,7 @@ function TransactionTableInner({
       : trans.error;
 
     const hasSplitError =
+      trans.is_parent &&
```

Comment:
Adding this as a fallback for safety: we shouldn't try to show errors on a transaction if it's not actually
a split.